src/content/self-hosted/install/releases.mdx

```diff
@@ -11,6 +11,7 @@ TBD
 ### Features
 
 - Support for the annotation [dev.okteto.com/endpoints](cloud/ssl.mdx#customizing-the-endpoints-shown-in-the-okteto-ui) to customize the endpoints shown in the Okteto UI.
+- Support for [token-based authentication](self-hosted/administration/configuration.mdx#token) to facilitate testing scenarios.
```

Comment:
Maybe an obvious question: why is token-based auth specifically good for facilitating test scenarios? My assumption is that auth providers don't really have a good experience for test scenarios so token-based auth allows devs to emulate that process. Is it worth articulating this (or the correct rationale) in docs for better clarity?

I'm thinking about people not familiar with Okteto already, devs who maybe aren't familiar with this nuance, and/or maybe more junior devs who just don't understand the dynamics yet.

Reply:
Configuring other auth methods is more time-consuming and it usually requires approvals from other teams. What I mean here is, that if you want to give it a try to okteto, it's way easier to configure a token for a single user, login to the Okteto UI, and get a sense of our platform. Then, if you like it, it's recommended to use another auth method, which are more secure and they also support multiuser. Token-based is to make it easier to experiment with the Okteto Platform.
How would you phrase that?

Reply:
@pchico83 I think that's a perfect explanation of why we're adding this 👍 Maybe we can add something like that to your proposed `Token` section in `/self-hosted/administration/configuration.mdx#token`. My proposal:

Current language you have in this commit:

> Token-based is the default authentication method.
> This method permits users to validate their identity by employing a randomly generated token comprising eight characters.
> The token-based authentication approach is an optimal choice for testing/evaluation scenarios, where access to Okteto is restricted to a single user. This is not recommended for multiple users or production setups.

My suggestion:

> Okteto defaults to token-based authentication to simplify the installation and configuration of Okteto self-hosted.
> 
> This enables you to configure a token for a single user, log in to the Okteto UI, and start exploring Okteto's features and functionality much faster than configuring IAM tools. Configuring other authentication tools typically requires approvals from other teams and are more time-consuming and cumbersome to configure.
> 
> Once you've completed the initial configuration of Okteto using token-based authentication and explored the product, you can then switch to a more optimal IAM solution. Token-based authentication would no longer be ideal as you expand to multiple users and incorporating production environments into Okteto.

WDYT? This way the documentation can be a bit more direct, educational, and purposeful in explaining why we've introduced a feature 😄

Reply:
That sounds amazing, thanks @omnomagonz !